reporter/build.gradle.kts

```diff
@@ -23,6 +23,7 @@ val apachePoiSchemasVersion: String by project
 val cyclonedxCoreJavaVersion: String by project
 val flexmarkVersion: String by project
 val hamcrestCoreVersion: String by project
+val jacksonVersion: String by project
```

Comment:
commit-msg: Is the ORT evaluator also envisioned to be a future user of this model?

Reply:
Not at this time, because the evaluator output is used as input for the model.